### PR TITLE
Enhancement - Find Command

### PR DESCRIPTION
**Pull request by melvintzw**
**Enhancement to Existing Feature**
Enhance FIND command so as to search through a person's address and tags

- edit PersonContainsKeywordsPredicate.java (a Predicate) in model.person to look through the tags and address of the Person instance
- add a helper method, getStringOfTags() to decompose tags into a string of words to check against

Add test cases to test this new functionality

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package seedu.address.logic.commands;
 
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.PersonContainsKeywordsPredicate;
```
Comment (jonleeyz) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:3]: HAHA just wondering, why did you have to change it from `Name...` to `Person...`?

Comment (jonleeyz) [reply src/main/java/seedu/address/logic/commands/FindCommand.java:3]: Oh ok never mind, I realised why LOL.

**Review by jonleeyz — APPROVED** [review]
Implementation looks great, but I'll take a closer look at the JUnit Tests some time.

Thanks @melvintzw!

File: src/main/java/seedu/address/logic/commands/FindCommand.java (hunk 1 above)
Comment (melvintzw) [reply src/main/java/seedu/address/logic/commands/FindCommand.java:3]: haha yup yup

**Pull request merged**